edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ServiceProxy.cs

```diff
@@ -12,21 +12,25 @@ namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
 
     public class ServiceProxy : IServiceProxy
     {
-        readonly IDeviceScopeApiClient securityScopesApiClient;
+        readonly IDeviceScopeApiClientProvider securityScopesApiClientProvider;
+        readonly IDeviceScopeApiClient defaultSecurityScopesApiClient;
 
-        public ServiceProxy(IDeviceScopeApiClient securityScopesApiClient)
+        public ServiceProxy(IDeviceScopeApiClientProvider securityScopesApiClientProvider)
         {
-            this.securityScopesApiClient = Preconditions.CheckNotNull(securityScopesApiClient, nameof(securityScopesApiClient));
+            this.securityScopesApiClientProvider = Preconditions.CheckNotNull(securityScopesApiClientProvider, nameof(securityScopesApiClientProvider));
+            this.defaultSecurityScopesApiClient = this.securityScopesApiClientProvider.CreateDeviceScopeClient();
```

Comment:
if IDeviceScopeApiClientProvider knows only to CreateDeviceScopeClient and based in implementation return one or the other we don't need this